settings/common.py

```diff
@@ -23,6 +23,7 @@
 SITE_VARIABLES = {
```

Comment:
https://github.com/theskumar/python-dotenv is developed for the same reason to reduce the differences. It works great with ansisble/puppet or any other system. Having `django-configurations` and `fabric` is added advantage. Just my 2 cents! :blush: